cfg/nos3-mission.xml

```diff
@@ -1,8 +1,12 @@
 <nos3-mission-cfg>
     <start-time>814048200.0</start-time>
-    <gsw>cosmos</gsw>
+    <!-- <gsw>cosmos</gsw>
+    <fsw>cfs</fsw> -->
+    <gsw>fprime</gsw>
+    <fsw>fprime</fsw>
     <number-spacecraft>1</number-spacecraft>
-    <sc-1-cfg>sc-full-config.xml</sc-1-cfg>
+    <!-- <sc-1-cfg>sc-full-config.xml</sc-1-cfg> -->
+    <sc-1-cfg>sc-minimal-config.xml</sc-1-cfg>
```

Comment:
Should include comments that F' works with only the minimal configuration presently (if that's true)

Reply:
Noted, will add comments and make default cfs cosmos configuration.

Reply:
commited changes